waspc/cli/src/Wasp/Cli/Command/CreateNewProject.hs

```diff
@@ -6,18 +6,18 @@ where
 import Control.Monad.Except (throwError)
 import Control.Monad.IO.Class (liftIO)
 import Data.List (intercalate)
-import StrongPath (Abs, Dir, File', Path', Rel, reldir, relfile, (</>))
+import StrongPath (Abs, Dir, Path, Path', System, reldir, relfile, (</>))
 import qualified StrongPath as SP
-import System.Directory (createDirectory, getCurrentDirectory)
-import qualified System.Directory
+import System.Directory (getCurrentDirectory)
 import qualified System.FilePath as FP
 import Text.Printf (printf)
 import Wasp.Analyzer.Parser (isValidWaspIdentifier)
-import Wasp.AppSpec.ExternalCode (SourceExternalCodeDir)
 import Wasp.Cli.Command (Command, CommandError (..))
 import qualified Wasp.Cli.Command.Common as Command.Common
+import Wasp.Cli.Common (WaspProjectDir)
 import qualified Wasp.Cli.Common as Common
-import qualified Wasp.Data
+import qualified Wasp.Data as Data
+import Wasp.Generator.FileDraft.WriteableMonad (WriteableMonad (copyDirectoryRecursive))
```

Comment:
Crap, I missed that it was in `Wasp.Generator`. Good thing I made sure to [generalize `WritetableMonad`](https://github.com/wasp-lang/wasp/pull/420) with RankNTypes because I needed it here :sweat_smile:

Reply:
Hey you still improved it which is nice :D!